lib/grpc_web/client/client_executor.rb

```diff
@@ -81,13 +81,18 @@ def parse_headers(header_str)
       headers
     end
 
-    def raise_on_response_errors(resp, headers, error_unpacking_frames)
-      status_str = headers[GRPC_STATUS_HEADER]
+    def raise_on_response_errors(resp, metadata, error_unpacking_frames)
+      metadata.delete('x-grpc-web')
```

Comment:
It's a nitpick but I think it's worth creating a copy of the hash to avoid modifying the supplied hash. Less room for confusion at least. Could do by `dup`ing the hash or using `.reject` to generate a new subset.

Reply:
fixed in #9fd79d7